### PR TITLE
DWPF-1143 Updates cyber software form

### PR DESCRIPTION
**Pull request by nicopicchio**

### CODE REVIEW

**Comment by nicopicchio** [comment]
TODO => Add Hint text to options

File: core/migrations/0160_cyber_software_data_update.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+from django.db import migrations
+
+
+REFERENCE_DATA_MODEL = "CyberSoftware"
+
+
+def forwards(apps, schema_editor):
+    target_model = apps.get_model("core", REFERENCE_DATA_MODEL)
+    # check with Sam about existing IRAP reqs before deleting the below options
```
Comment (SamDudley) [inline core/migrations/0160_cyber_software_data_update.py:11]: You can remove this comment now.

**Review by SamDudley — APPROVED** [review]
Top job!

File: core/migrations/0160_cyber_software_data_update.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    accessing_website_obj = target_model.objects.get(code="accessing_website")
+    accessing_website_obj.name = "Accessing a website to download or access information"
+    accessing_website_obj.ordering = 3
+    accessing_website_obj.save(update_fields=["name", "ordering"])
+
+    target_model.objects.create(
+        code="building_new_software",
+        name="Building a new service or software",
+        ordering=2,
+    )
```
Comment (SamDudley) [inline core/migrations/0160_cyber_software_data_update.py:52]: You can remove this now as it isn't in your `forward` method.

**Review by SamDudley — APPROVED** [review]

**Pull request merged**